src/query_parser/select_statement.py

```diff
@@ -0,0 +1,63 @@
+from src.query_parser.eva_statement import EvaStatement
+from src.query_parser.types import StatementType
+from src.expression.abstract_expression import AbstractExpression
+from src.loaders.abstract_loader import AbstractLoader
+from src.query_parser.table_ref import TableRef
+from typing import List
+
+
+class SelectStatement(EvaStatement):
+    """
+    Select Statement constructed after parsing the input query
+
+    Attributes
+    ----------
+    _target_list : List[AbstractExpression]
+        list of target attributes in the select query,
+        each attribute is represented as a Abstract Expression
+    _from_table : TableRef
+        table reference in the select query
+    _where_clause : AbstractExpression
+        predicate of the select query, represented as a expression tree.
+    **kwargs : to support other functionality, Orderby, Distinct, Groupby.
+    """
+
+    def __init__(self, target_list=None, from_table=None,
```

Comment:
Let's try to use type annotations as much as possible for clarity. 
ex: target_list: List[AbstractExpression]